drivers/health_pctp/app/models/document_exports/health_pctp_pdf_export.rb

```diff
@@ -0,0 +1,69 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module DocumentExports
+  class HealthPctpPdfExport < ::GrdaWarehouse::DocumentExport
```

Comment:
Since this is healthcare document, it gets stored in the healthcare db
```suggestion
  class HealthPctpPdfExport < ::Health::DocumentExport
```